main.cpp

```diff
@@ -0,0 +1,49 @@
+#include <iostream>
+#include <string>
+#include <exception>
+#include <boost/asio.hpp>
+// #include "nginx-configparser/config_parser.h"
+
+// Inspired by https://github.com/egalli64/thisthread/blob/master/asio/tcpIpCs.cpp
+void runTcpServer(int port_number) {
+	boost::asio::io_service aios;
+	boost::asio::ip::tcp::acceptor acceptor(aios, boost::asio::ip::tcp::endpoint(boost::asio::ip::tcp::v4(), port_number));
```

Comment:
Docs online seem to show that this is the standard way of invoking the boost acceptor method, I think we can keep it this way. I'll reformat each parameter on a new line for readability.